### PR TITLE
Update dsn_levels.py v.0.1.7

### PR DESCRIPTION
**Pull request by shoc71**
added level mouse with new patch changes taken into account

### CODE REVIEW

File: dont stop now/dsn_levels.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ def update(self):
         if self.player.alive and not self.player.freeze and \
                 not self.level_condition:
             self.deaths += self.player.death(self.death_zones)
-            self.player.collision_plat(self.platforms)
+            self.player.collision_plat(self.platforms + self.walls)
```
Comment (slysnivy) [inline dont stop now/dsn_levels.py:88]: self.walls is recommended to be apart of platform collision for proper collision

File: dont stop now/dsn_levels.py [hunk 2]
```diff
@@ -210,7 +210,7 @@ def update(self):
         LevelScene.update(self)
         self.player.alive = True
         self.victory_counter = len(self.victory_text)
-        if (random.randint(1, 2500) <= 10) and not self.player.enable_gravity:
```
Comment (slysnivy) [inline dont stop now/dsn_levels.py:213]: This should be 15 to increase the chance of jumping for the menu square

**Review by slysnivy — CHANGES_REQUESTED** [review]
Refer to the 2 review comments and update these values for this script

**Comment by shoc71** [comment]
both those requested changes are in this new patch in lines code 88 and 213. I believe you are referring v.0.1.6 dsn_levels.py which do not have what you have requested.

**Pull request merged**